hpedockerplugin/hpe/hpe_3par_mediator.py

```diff
@@ -1033,9 +1033,53 @@ def create_vfs(self, vfs_name, ip, subnet, cpg=None, fpg=None,
         finally:
             self._wsapi_logout()
 
-    def set_ACL(fUName, fGName, fMode):
-        LOG.info("Inside set ACL call but temperory not making"
-                 " any rest call.")
+    def set_ACL(self, fMode, fUserId, fUName, fGName):
+        # fsMode = "A:fdps:rwaAxdD,A:fFdps:rwaxdnNcCoy,A:fdgps:DtnNcy"
+        ACLList = []
+        per_type = {"A": 1, "D": 2, "U": 3, "L": 4}
+        fsMode_list = fMode.split(",")
+        principal_list = ['OWNER@', 'GROUP@', 'EVERYONE@']
+        for index, value in enumerate(fsMode_list):
+            acl_values = value.split(":")
+            acl_type = per_type.get(acl_values[0])
+            acl_flags = acl_values[1]
+            acl_principal = ""
+            if index == 0:
+                acl_principal = principal_list[0]
+            if index == 1:
+                acl_principal = principal_list[1]
+            if index == 2:
+                acl_principal = principal_list[2]
+            acl_permission = acl_values[2]
+            acl_object = {}
+            acl_object['aclType'] = acl_type
+            acl_object['aclFlags'] = acl_flags
+            acl_object['aclPrincipal'] = acl_principal
+            acl_object['aclPermissions'] = acl_permission
+            ACLList.append(acl_object)
+            LOG.info("Inside set ACL call but temperory not making"
+                     " any rest call.")
```

Comment:
Looks like this message crept in from earlier code when we had just a place holder for this function with this log message :)
Can we modify it to capture better what we are doing here or remove it?

Reply:
Removed. Done